cloudformation/media-atom-maker.json

```diff
@@ -236,6 +236,48 @@
                 ]
             }
         },
+        "UserUploadBucket": {
+            "Type": "AWS::S3::Bucket",
+            "Properties": {
+                "CorsConfiguration": {
+                    "CorsRules": [{
+                        "AllowedHeaders": ["*"],
+                        "ExposedHeaders": ["ETag"],
+                        "AllowedMethods": ["GET", "PUT", "POST", "DELETE"],
+                        "AllowedOrigins": [
+                          { "Fn::Join": ["", [ "https://", { "Ref": "DomainToMonitor" } ]] }
+                        ]
+                    }]
+                }
+            }
+        },
+        "LimitedUploadRole": {
+            "Type": "AWS::IAM::Role",
+            "Properties": {
+                "AssumeRolePolicyDocument": {
+                    "Statement": [{
+                        "Effect": "Allow",
+                        "Principal": {
+                            "AWS": { "Fn::Join": ["", [ "arn:aws:iam::", { "Ref": "AWS::AccountId" }, ":root"]] }
+                        },
+                        "Action": "sts:AssumeRole"
+                    }]
+                },
+                "Path": "/",
+                "Policies": [
+                    {
+                        "PolicyName": "LimitedUploadPolicy",
+                        "PolicyDocument": {
+                            "Statement": [{
+                                "Effect": "Allow",
+                                "Action": ["s3:PutObject", "s3:PutObjectAcl", "s3:ListBucketMultipartUploads", "s3: ListMultipartUploadParts", "s3:AbortMultipartUpload"],
```

Comment:
Small one but could we put the Action list on new lines to make it easier to read?